### PR TITLE
Added CTRL+H shortcut to toggle hidden folder visibility. (#6867)

### LINKED ISSUE #6867: Add a keyboard shortcut to show hidden files
### What's the Problem?

_No response_

### Solution/Idea

I think it would be a pretty good idea to have a shortcut to toggle if hidden files are show. I use this a lot in Linux and would be awesome to have this in Files too! Some suggestions of key binds would be "ctrl + h" or "ctrl + shift + ."

### Alternatives

To be honest, I don't know what would be an alternative.

### Priorities

| Capability | Priority |
| :---------- | :------- |
| This proposal will allow developers to accomplish W | Must |
| This proposal will allow end users to accomplish X | Should |
| This proposal will allow developers to accomplish Y | Could |
| This proposal will allow end users to accomplish Z | Won't |

### Files Version

2.0.33

### Windows Version

Windows 11 21H2 22000.282

### Additional comment

_No response_

### PR DESCRIPTION
**Pull request by QuaintMako**
**Resolved / Related Issues**
- Closes #6867 

**Details of Changes**
- Added the CTRL+H shortcut to toggle the hidden files visibility.

**Validation**
How did you test these changes?
- [X] Built and ran the app
- [ ] Tested the changes for accessibility

### CODE REVIEW

**Comment by QuaintMako** [comment]
This is my first PR ever, please let me now if I need to do anything more! :)

Just a foot note, the code produced has not been copied from the previous solution, as required in the issue discussions.

File: src/Files.Uwp/Views/ModernShellPage.xaml.cs [hunk 1]
```diff
@@ -848,6 +848,11 @@ private async void KeyboardAccelerator_Invoked(KeyboardAccelerator sender, Keybo
                 case (true, true, false, _, VirtualKey.Number7): // ctrl+shift+7, adaptive
                     InstanceViewModel.FolderSettings.ToggleLayoutModeAdaptive();
                     break;
+
+                case (true, false, false, true, VirtualKey.H): // ctrl + h, toggle hidden folder visibility
+                    var hiddenItemCurrentVisibitily = UserSettingsService.PreferencesSettingsService.AreHiddenItemsVisible;
```
Comment (jiejasonliu) [inline src/Files.Uwp/Views/ModernShellPage.xaml.cs:853]: typo: `hiddenItemCurrentVisibility`

nit: can we move this closer to where the non-tab conditions are? somewhere before the `F1` shortcut should be sufficient

**Comment by QuaintMako** [comment]
Thanks for taking care of the changes @gave92 , couldn't take care of them right away!

**Comment by gave92** [comment]
No problem, and thanks for this PR!

**Review by gave92 — APPROVED** [review]

**Pull request merged**